InvenTree/InvenTree/settings.py

```diff
@@ -912,3 +913,21 @@ def _is_true(x):
 PLUGIN_TESTING_SETUP = get_setting('PLUGIN_TESTING_SETUP', False)  # load plugins from setup hooks in testing?
 PLUGIN_RETRY = get_setting('PLUGIN_RETRY', 5)  # how often should plugin loading be tried?
 PLUGIN_FILE_CHECKED = False                    # Was the plugin file checked?
+
+# user interface customization values
+CUSTOMIZE = get_setting(
+    'INVENTREE_CUSTOMIZE',
+    CONFIG.get('customize', {}),
+    {}
+)
+
+CUSTOM_LOGO = get_setting(
+    'INVENTREE_LOGO',
+    CONFIG.get('logo', False),
```

Comment:
I would suggest that the custom logo should be specified in the "customize" section of the config file, making this line

```python
CUSTOM_LOGO = get_setting(
    'INVENTREE_CUSTOM_LOGO',
    CUSTOMIZE.get('logo', False)
)
```

Then, the `config.yaml` file could look like:

```yaml
customize:
  navbar_message: My custom navbar message
  login_message: My custom login message
  logo: media/my_custom_logo.png
```